src/tasks/AppleAppBuilder/AppleAppBuilder.cs

```diff
@@ -27,7 +27,7 @@ public string TargetOS
 
         set
         {
-            targetOS = value.ToLower();
+            targetOS = value.ToLowerInvariant();
```

Comment:
@radical, can you confirm the changes in this file and the one below are appropriate?